src/Core/Verbs/Verb.fs

```diff
@@ -23,3 +29,5 @@ let hasRegularParticiple word =
 let getParticiplePattern = 
     ParticiplePatternDetector.getPattern
 
+let getRandomPronoun () = 
```

Comment:
Braces likely not needed
```suggestion
let getRandomPronoun = 
```

Reply:
if no braces it will be a value - not a function

Reply:
Right, this was off